packages/core/contracts/cross-chain-oracle/OracleSpoke.sol

```diff
@@ -22,29 +23,25 @@ import "./interfaces/ChildMessengerConsumerInterface.sol";
  */
 contract OracleSpoke is
     OracleBase,
+    SpokeBase,
     OracleAncillaryInterface,
     OracleInterface,
     ChildMessengerConsumerInterface,
     Lockable
 {
-    ChildMessengerInterface public messenger;
-
-    event SetChildMessenger(address indexed childMessenger);
-
-    constructor(address _finderAddress, ChildMessengerInterface _messengerAddress) OracleBase(_finderAddress) {
-        messenger = _messengerAddress;
-        emit SetChildMessenger(address(messenger));
-    }
+    constructor(address _finderAddress) OracleBase(_finderAddress) SpokeBase(_finderAddress) {}
 
     // This assumes that the local network has a Registry that resembles the mainnet registry.
     modifier onlyRegisteredContract() {
         RegistryInterface registry = RegistryInterface(finder.getImplementationAddress(OracleInterfaces.Registry));
         require(registry.isContractRegistered(msg.sender), "Caller must be registered");
         _;
     }
-
-    modifier onlyMessenger() {
-        require(msg.sender == address(messenger), "Caller must be messenger");
+    modifier onlyGovernorSpoke() {
+        require(
+            msg.sender == finder.getImplementationAddress(OracleInterfaces.GovernorSpoke),
+            "Caller must be governor spoke"
+        );
```

Comment:
I may be totally missing something, but where is this used?

Reply:
it's not! good catch. removed.